src/GovUk.Education.ExploreEducationStatistics.Content.Model/Database/ContentDbContext.cs

```diff
@@ -168,6 +169,11 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 .Property(b => b.Status)
                 .HasConversion(new EnumToStringConverter<MethodologyStatus>());
 
+            modelBuilder.Entity<MethodologyFile>()
+                .HasOne(r => r.Methodology)
+                .WithMany()
+                .OnDelete(DeleteBehavior.NoAction);
```

Comment:
This is different to what is in the migration itself?

Reply:
The relationship in the generated code (check carefully, it's the foreign key `FK_MethodologyFiles_Methodologies_MethodologyId` is actually not specified, and the default in `Microsoft.EntityFrameworkCore.Migrations.Operations.Builders.ForeignKey` if not specified is `DeleteBehaviour.NoAction`.